### PR TITLE
Update driver a34401 a

### PR DESCRIPTION
**Pull request by Bachibouzouk**

### CODE REVIEW

**Comment by taliamo** [comment]
Testing more functions with HP34401A and A34401A (hewlett packard vs agilent) and respective drivers "hp34401" and "a34401" --seems that hp34401 driver works for the Agilent machine and a34401 driver works for neither. I suggest demarcating somehow, that folks should use the HP34401 driver file going forward. ~\^^/~

**Comment by Bachibouzouk** [comment]
>  I suggest demarcating somehow, that folks should use the HP34401 driver file going forward. ^^/

@taliamo - We could simply remove the A34401A file.

File: LabDrivers/HP34401A.py [hunk 1]
```diff
@@ -94,8 +94,9 @@ def read_resistance(self):
             return 123.4
 
     # if run as own program
-    # if (__name__ == '__main__'):
-
+if (__name__ == '__main__'):
```
Comment (Bachibouzouk) [inline LabDrivers/HP34401A.py:97]: Usually we don't push the changes made within `if (__name__ == '__main__'):`

**Review by Bachibouzouk — COMMENTED** [review]
@taliamo : if there is no change to the driver file we can simply acknowledge the fact that you tested the driver in the issue #10 (one could make a checklist with all drivers name within the issue's conversation thread)

**Comment by Bachibouzouk** [comment]
This is no longer up-to-date

**Pull request closed without merging**